deployment/Dockerfile.debian.amd64

```diff
@@ -0,0 +1,31 @@
+FROM debian:10
```

Comment:
I don't think so. It's been almost a year since 10 was released, and the SSL versions are incompatible. This is a forward-port of the previous build infra anyways which was Deb10.